### ISSUE TITLE
Any update on how to use a pre-trained VQ-rec for real time recommendation ?

### ISSUE THREAD

**Issue by shamanez** [RUCAIBox/VQ-Rec#4]
In VQ-rec, we use text to represent both user and item spaces.  In my use case, I get items that are new and have zero interactions. 

So I wonder is there a way, I can use a pre-trained VQ-rec model to recommend new items to already available users (I have their most recent interactions with other items)

**Comment by hyp1231** [comment]
Yes but expected to have one script uploaded in early May due to the limited bandwidth. Sorry for waiting.

**Comment by shamanez** [comment]
Thanks
